src/lib/output/SarifOutputFormatter.ts

```diff
@@ -304,7 +304,7 @@ class RetireJsSarifFormatter extends SarifFormatter {
 					driver: {
 						name: 'Retire.js',
 						// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-						version: retire.version as string,
+						version: retire.version,
```

Comment:
yet another new eslint rule... doesn't like casting to the type that it already knows (in this case string)